examples/pub/lib/pub_repository.dart

```diff
@@ -7,15 +7,21 @@ part 'pub_repository.g.dart';
 
 class PubRepository {
   final dio = Dio();
+  final scheme="https";
+  final host='pub.dartlang.org';
+  final packagesPath='api/packages';
+  final searchPath='api/search';
+  final likesPath='api/account/likes';
```

Comment:
I wouldn't have those constants, as these are specific to the API used. Some APIs have parts of their path in common, but that's more of an implementation detail than something we really want to share